en/docs/guides/identity-lifecycles/self-registration-workflow.md

```diff
@@ -4,9 +4,253 @@ This page guides you through setting up self-registration for users to onboard t
 
 ----
 
-## Self register using the user portal 
+## Self register using My Account
 
-TODO: user-portal-fragment
+??? Warning "Click to see instructions specific for a migrated deployment" 
+    If you have migrated from a previous WSO2 Identity Server version, ensure that the `event.default_listener.identity_mgt` with the
+    `         priority=50        ` is set to **false** and that the Identity
+    Listeners with `         priority=95        ` and
+    `         priority=97        ` are set to **true** in the
+    `         <IS_HOME>/repository/conf/deployment.toml        ` file.
+   
+    !!! Note 
+        If the following configurations are not present in the `deployment.toml ` file already, you do not have to add it. 
+    
+    ``` java
+    [event.default_listener.identity_mgt]
+    priority= "50"
+    enable = false
+    [event.default_listener.governance_identity_mgt]
+    priority= "95"
+    enable = true
+    [event.default_listener.governance_identity_store]
+    priority= "97"
+    enable = true
+    ```
+
+Follow the steps given below to register users for the super tenant,
+which is `         carbon.super        `.
+
+
+1.  Enable the email sending configurations of the WSO2 Identity Server
+    as explained [here](../../../deploy/configure-email-sending/).
```

Comment:
since this is a mandatory step, shouldn't we have this content inline?
can use the configure-email-sending.md fragment